src/main/java/com/google/devtools/build/lib/bazel/bzlmod/SingleExtensionUsagesValue.java

```diff
@@ -56,13 +57,16 @@ public abstract class SingleExtensionUsagesValue implements SkyValue {
   /** The repo mappings to use for each module that used this extension. */
   public abstract ImmutableMap<ModuleKey, RepositoryMapping> getRepoMappings();
 
+  public abstract ImmutableMap<String, RepositoryName> getRepoOverrides();
```

Comment:
add javadoc -- particularly, what do the key and value types mean (this is less obvious than the same field in `MdouleExtensionUsage`)